Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -36,7 +36,7 @@ void GridFormattingContext::run(Box const& box, LayoutMode)
     };
 
     auto maybe_add_column_to_occupation_grid = [](int needed_number_of_columns, Vector<Vector<bool>>& occupation_grid) -> void {
-        int current_column_count = (int)occupation_grid[0].size();
+        int current_column_count = static_cast<int>(occupation_grid[0].size());
```

Comment:
I would consider making `OccupationGrid` it's own little class with an API like (`.column_count()`) that returns the count as an `int` so you don't need to do all these casts.

Reply:
I really liked this idea :D